src/Symfony/Component/HttpKernel/Controller/ArgumentResolver.php

```diff
@@ -12,6 +12,11 @@
 namespace Symfony\Component\HttpKernel\Controller;
 
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Controller\ArgumentValueResolver\DefaultValueResolver;
```

Comment:
Is the term "value" really needed in namespace and classes ?

It could be:
```php
..\ArgumentResover\DefaultArgumentResolver
..\ArgumentResover\RequestArgumentResolver
..\ArgumentResover\VariadicArgumentResolver
...
```
?